packages/gasket-nextjs/package.json

```diff
@@ -43,7 +43,9 @@
   "homepage": "https://github.com/godaddy/gasket/tree/main/packages/gasket-nextjs",
   "dependencies": {
     "@gasket/data": "^6.41.1",
-    "htmlescape": "^1.1.1"
+    "htmlescape": "^1.1.1",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0"
```

Comment:
`react` should continue to be a devDep to allow control of version at the app level. I would say same `prop-types`.

Reply:
100% agree those should both stay as devDep

Reply:
Awesome. Updates have been made. Appreciate the reviews!